perimeterx/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "display_name": "PerimeterX",
+  "maintainer": "support@perimeterx.com",
+  "manifest_version": "1.0.0",
+  "name": "perimeterx",
+  "metric_prefix": "perimeterx.",
+  "metric_to_check": "",
+  "creates_events": true,
+  "short_description": "Integrate PerimeterX Logs and Metrics with DataDog",
+  "guid": "6e3a9bc2-6766-4b24-9edf-12811d821d41",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-perimeterx Integration",
+  "categories": [
+    "security"
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "perimeterx",
+  "assets": {
+    "dashboards": {},
+    "saved_views": {},
+    "monitors": {},
+    "service_checks": "assets/service_checks.json"
+  },
+  "is_beta": true
```

Comment:
I think this is ok for now since we're still building out our integration.

Reply:
After some discussion, I think I now agree with you that we can change this to _false_ since we appear to be very close to having our integration ready